src/platform_impl/web/web_sys/mod.rs

```diff
@@ -15,6 +18,10 @@ pub fn throw(msg: &str) {
     wasm_bindgen::throw_str(msg);
 }
 
+pub fn log(msg: &str) {
+    web_sys::console::log_1(&msg.into());
+}
+
```

Comment:
Is this a debug convenience left here on accident?

Reply:
Yes, I can remove it